src/openlcb/MemoryConfigClient.hxx

```diff
@@ -529,8 +538,17 @@ private:
         if (responseCode_ & DatagramClient::OPERATION_PENDING)
         {
             isWaitingForTimer_ = 1;
+            // Retrieve the timeout flags
```

Comment:
can/should we consider moving this to an inline method to reduce duplication of this block of code?